modules/release-notes/pages/0.6.24-rn.adoc

```diff
@@ -21,14 +21,44 @@ The most significant new features and capabilities include updates in the follow
 
 === DFX
 
-* A new `+deploy-wallet+` subcommand enables you to specify the canister identifier for your cycles wallet WebAssembly module (WASM).
+* With this release, you must have a **wallet canister** with **cycles** to deploy or manage applications on the {IC}.
++
+For local development, `+dfx+` automatically creates a wallet for you when you run `+dfx canister create+` or `+dfx deploy+` commands within each project.
++
+To deploy and manage applications on the main {IC} network, you must provide the {company-id} Foundation with a principal associated with your user identity and receive a **wallet canister identifier** in return.
++
+To deploy applications on the {IC} network:
++
+--
+. Download and install the {sdk-short-name}.
+.  Run the `dfx identity get-principal` command to create your default identity and principal:
++
```

Comment:
We also automatically create a wallet for people on Sodium. How about we just say we've begun introducing wallets  but there is no cycle accounting on Sodium. Therefore, the wallet will just be a pass through facilitator for the dfx canister commands.